apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -257,6 +280,38 @@ export class SocketServer implements IAdapter {
     }
   }
 
+  public sendRuntimeStoreUpdate(keysToUpdate: (keyof RuntimeStore)[], store: Partial<RuntimeStore>) {
+    // create a patch object with all the keys marked for updating
+    const patch = {};
+    keysToUpdate.map((key) => {
+      Object.assign(patch, { [key]: store[key] });
+    });
+
+    //convert to JSON once and reuse
+    const stringifiedPatch = JSON.stringify({ type: 'ontime-patch', payload: patch });
+
+    // for each client that have subscribed to the patch method send the patch object
+    this.patchClients.forEach((ws) => {
```

Comment:
do you think this is necessary?

I understand the motivation behind it, but I wonder if trying to keep backwards compatibility makes our code harder and more bug prone

I am ok with this as long as we are not trading the above for the compatibility

Reply:
We would have to do a breaking version change right?
I would love to keep it simple, but I have no idea what things that are dependent one us?

Reply:
what if we continue sending the messages in serial?
the logic of the store should accumulate changes in the same event loop iteration, maybe the small difference in milliseconds on the message being received and parsed will not be visible?

Reply:
hmm yeah it's worth a try

Reply:
That seamed to work